doc/make.bat

```diff
@@ -11,7 +11,7 @@ if "%SPHINXBUILD%" == "" (
 )
 set SOURCEDIR=source
 set BUILDDIR=_build
-set SPHINXOPTS=-j auto -W --keep-going -w build_errors.txt -N -q
+set SPHINXOPTS=-j auto -w build_errors.txt -N -q
```

Comment:
Is it correct that we'll reinstate the warnings after we fix all warnings coming from pyfluent source?

Reply:
Yeah, I think he has to have them off for now because there are many undocumented functions that show up in the new structure.  When I originally created the documentation I just left out any modules that did not contain any documentation at all.  So, one must either 1. Leave them undocumented and not showing up in the Sphinx doc at all or 2. document them.

Reply:
Yes we will discuss these suggestions. Thank you.